### DISCUSSION TITLE
What exactly are the steps to setting up something like the Screen Mirror example in the demo?

### DISCUSSION THREAD

**Discussion by kainbryanjones** [yume-chan/ya-webadb#612] [Q&A]
Cool library. We used it earlier this year to build a tool for our android app.
I'm now researching how we can add something like the screen mirror example.
Could I please just get some steps into how to do this?
Is there a guide somewhere?

**Comment by yume-chan** [comment]
There are some instructions in the README:

* https://github.com/yume-chan/ya-webadb/tree/main/libraries/scrcpy#readme
* https://github.com/yume-chan/ya-webadb/tree/main/libraries/adb-scrcpy#readme
